augur/augur_client.py

```diff
@@ -53,8 +56,10 @@ def __init__(self, hostname, port):
     '''
     self._hostname = hostname
     self._port = port
-    self._node = None
+    self._augur = None
```

Comment:
Similarly, a variable named ```_augur``` gives no information about what this is. Augur is an entire decentralized prediction platform so calling a variable by that name gives a reader no context.